zuul.d/adoption.yaml

```diff
@@ -239,63 +239,87 @@
               external:
                 ip: 172.21.0.103
                 config_nm: false
-          overcloud-controller-1:
+          overcloud-controller-1: &multinode-crlr-leaf1
             networks:
               default:
                 ip: 192.168.122.104
                 config_nm: false
               internal-api:
-                ip: 172.17.0.104
+                ip: 172.17.1.104
                 config_nm: false
               storage:
-                ip: 172.18.0.104
+                ip: 172.18.1.104
                 config_nm: false
               tenant:
-                ip: 172.19.0.104
+                ip: 172.19.1.104
                 config_nm: false
               storage_mgmt:
-                ip: 172.20.0.104
+                ip: 172.20.1.104
                 config_nm: false
               external:
-                ip: 172.21.0.104
+                ip: 172.21.1.104
                 config_nm: false
-          overcloud-controller-2:
+          overcloud-controller-2: &multinode-crlr-leaf2
             networks:
               default:
                 ip: 192.168.122.105
                 config_nm: false
               internal-api:
-                ip: 172.17.0.105
+                ip: 172.17.2.105
```

Comment:
since you are adding a new base job below can you please keep these changes confined there?
i.e. avoid changes to cifmw-adoption-base-source-multinode and keep the changes in cifmw-adoption-base-source-multinode-novacells

Reply:
the plan is to reuse the same networks "leafs" for both jobs, to reduce the code duplication and to simplify maintaining changes across them (see in the commit message)

Reply:
i would prefer not to make more changes to the multinode config while we (me and @cescgina) are still working on that as already discussed.

If this is really a blocking issue for you then perhaps we can consider it but you would have to ensure this is working/doesn't break the existing job (i.e. please make sure you are running the periodic-data-plane-adoption-multinode-osp-17-to-extracted-crc-no-ceph at least, and lets coordinate with Joan regarding the ceph job when you are closer to merging this).

However for now, can you please mark this as draft to prevent accidental merge?

Reply:
forgot to add... note the

```
config_nm: false
```

which means the CI framework network code will not apply this configuration. For tripleo nodes, instead we have os-net-config, so you'll have to update that config instead to create the networks with the ranges you want https://github.com/openstack-k8s-operators/install_yamls/blob/c154ee06caecb856ad910979da218bd3b47f22e2/devsetup/tripleo/overcloud_net_config.j2#L2

Reply:
done